### PR TITLE
Add Moonbase Alpha Sample DApps Tutorials

### PR DESCRIPTION
**Pull request by themacexpert**
### Description
Adds brief tutorials for each of the Moonbase Alpha Sample DApps as well as a quick links section for DApp URLs and Github Repository URLs. Links to existing Moonbeam Docs site content are included where relevant. 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

PR to update image paths in CN repo: https://github.com/PureStake/moonbeam-docs-cn/pull/84

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: builders/get-started/networks/moonbase.md [hunk 1]
```diff
@@ -57,7 +57,93 @@ To get DEV tokens, enter the following message, replacing `<enter-address-here->
...
+
+The Moonbase ERC20 Minter enables you to mint a variety of ERC-20 test tokens corresponding to the 8 planets of the Solar System, and Pluto. To mint tokens, first press **Connect MetaMask** in the upper right hand corner. Then scroll to the Mint Tokens section and the choose desired ERC20 contract. Press **Submit Tx** and confirm the transaction in MetaMask. Each mint will grant you 100 tokens, and you can mint tokens for each contract once per hour.
+
+![ERC20 Minter](/images/builders/get-started/networks/moonbase/moonbase-3.png)
+
+### Moonbeam Uniswap {: #moonbeam-uniswap } 
+
+[Moonbeam Uniswap](https://moonbeam-swap.netlify.app/#/swap){target=_blank} is a fork of [Uniswap-V2](https://uniswap.org/blog/uniswap-v2){target=_blank} deployed to Moonbase Alpha. Notably, Moonbeam Uniswap allows developers to easily make a swap to acquire [cross chain assets](/builders/xcm/xc20/) such as xcKarura or xcUnits for XCM testing purposes. To perform your first swap, take the following steps:
+
+1. Connect your MetaMask Wallet and ensure you're on the Moonbase Alpha network
```
Comment (eshaben) [inline builders/get-started/networks/moonbase.md:95]: I think there are a few places where this line is copy pasta'd... wallet doesn't need to be capitalized here

**Review by eshaben — CHANGES_REQUESTED** [review]
Looks good! Mostly just minor nitpicking... haha sorry!

Also not sure about the images being pieced together, I think that could cause some confusion. For WalletConnect, there is already an image that you could re-use: https://docs.moonbeam.network/images/builders/integrations/wallets/walletconnect/walletconnect-2.png

But not sure about the other images like `moonbase-4.png` and `moonbase-8.png`

**Comment by eshaben** [comment]
Also not really related to this review, but just pointing it out here so I don't forget that the update to the `readme-banner` image is going to break the docs site when this gets merged. I need to fix it!

**Comment by themacexpert** [comment]
Thanks for the review @eshaben, this should be good to go now 👍

**Review by eshaben — APPROVED** [review]
LGTM 🚀

**Pull request merged**